docs/getting-started/integrations/angular.md

```diff
@@ -703,6 +703,45 @@ The `moduleResolution` option of the TypeScript configuration determines the alg
 * You can update Angular to version 18, where the `moduleResolution` option is set to `bundler`  by default.
 * You can import translations directly from our CDN, like: `import ‘https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/translations/es.umd.js’;`. This way, the editor will load the translations automatically, so you do not need to pass them manually into the config.
 
+### Potential issues with Jest testing
+
+Jest is the default test runner used by many Angular apps. Unfortunately, Jest does not use a real browser. Instead, it runs tests in Node.js with the use of JSDOM. JSDOM is not a complete DOM implementation and while it is apparently sufficient for standard apps, it is not able to polyfill all the DOM APIs that CKEditor&nbsp;5 requires.
+
+A better approach to test the component inside a fully-fledged web browser, implementing a complete DOM, is to use [`jest-puppeteer`](https://github.com/smooth-code/jest-puppeteer).
```

Comment:
Are we sure it will work? Would be a bummer to propose something that is not working.

Reply:
It'll work only in one specific scenario - initialization of editor and trying to enter some text to it. Nothing more. It's why I'm not sure if we should place this section in the docs. It's like we polyfill few or DOM methods from hundreds used by editor in hope to make basic scenario to work.

Reply:
I mean did we test `jest-puppeteer` works  without those over writes.

Reply:
Ah, nope. I was using it some time ago without any problems, but you are right, I'll confirm it.

Reply:
I removed it after all, while it's possible to test it using puppeteer, it turns out that it's harder to configure than setting up vite or playwright.